src/components/TrackTile/hooks/test/useRecentCodedValues.test.ts

```diff
@@ -43,11 +43,13 @@ describe('useTrackerValues', () => {
   it('should handle no saved data', async () => {
     getItem.mockReturnValue(undefined as never);
 
-    const { result, waitFor } = renderHook(() =>
+    const { result, waitForNextUpdate } = renderHook(() =>
       useRecentCodedValues('metricId'),
     );
 
-    await waitFor(() => expect(result.current).toEqual([]));
+    await waitForNextUpdate();
+
+    expect(result.current).toEqual([]);
   });
```

Comment:
Fixes this "not wrapped in act" error.

![Screenshot 2023-03-24 at 8 55 04 PM](https://user-images.githubusercontent.com/220893/227674684-ee97b048-17a3-4f60-9061-a3782f24f12b.png)